test/rules/std_L003_test.py

```diff
@@ -1,145 +1,6 @@
 """Tests the python routines within L003."""
-from unittest.mock import Mock
-
 import pytest
-
-from sqlfluff.core.config import FluffConfig
 from sqlfluff.rules.L003 import Rule_L003
-from sqlfluff.testing.rules import get_rule_from_set
-
-
-@pytest.mark.parametrize(
```

Comment:
Do you think we are losing anything important by removing these tests?

Reply:
No 100% not. The ones I took away are just checking internals that are well covered by the tests at large. Also they weren't clearly covering complex contained internal logic that needed to stay stable. just the input and outout of a large function.